run_service_ssh.ps1

```diff
@@ -1,8 +1,14 @@
 # This script first runs the ggp ssh init command to get the needed 
 # credentials to connect via ssh. Then it connects to the instance 
 # with portforwarding and starts OrbitService. To also deploy OrbitService,
-# invoke with a path to the OrbitService executable
-# e.g. run_service_ssh.ps1 build/bin/OrbitService  
+# invoke with a the argument -deploy and a path to the OrbitService executable
```

Comment:
Not sure what the convention on Windows is, but maybe this should be --deploy?

Reply:
See my comment above